### PR TITLE
Cleanup old ItContract properties

### PR DESCRIPTION
**Pull request by JacobMalling**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Core.ApplicationServices/ExcelService.cs [hunk 1]
```diff
@@ -717,7 +711,7 @@ private static DataTable GetItContractTable(IEnumerable<ItContract> itContracts)
                 var irrevocableTo = contract.IrrevocableTo.HasValue ? contract.IrrevocableTo.Value.ToShortDateString() : null;
                 var expirationDate = contract.ExpirationDate.HasValue ? contract.ExpirationDate.Value.ToShortDateString() : null;
                 var terminated = contract.Terminated.HasValue ? contract.Terminated.Value.ToShortDateString() : null;
-                table.Rows.Add(contract.Id, contract.Name, contract.ItContractId, contract.Esdh, contract.Folder, contract.Note, concluded, irrevocableTo, expirationDate, terminated);
+                table.Rows.Add(contract.Id, contract.Name, contract.ItContractId, contract.Note, concluded, irrevocableTo, expirationDate, terminated);
```
Comment (mrjsawdk) [inline Core.ApplicationServices/ExcelService.cs:714]: er det noget custom sql den laver her eller hvad ender det med?

File: Core.ApplicationServices/ExcelService.cs [hunk 2]
```diff
@@ -139,13 +139,11 @@ private IEnumerable<ExcelImportError> ImportItContractsTransaction(DataTable con
...
                     Name = row.Field<string>(1),
                     ItContractId = row.Field<string>(2),
-                    Esdh = row.Field<string>(3),
-                    Folder = row.Field<string>(4),
-                    Note = row.Field<string>(5),
-                    ConcludedText = row.Field<string>(6),
-                    IrrevocableToText = row.Field<string>(7),
-                    ExpirationDateText = row.Field<string>(8),
-                    TerminatedText = row.Field<string>(9),
+                    Note = row.Field<string>(3),
```
Comment (mrjsawdk) [inline Core.ApplicationServices/ExcelService.cs:142]: Hvad med ContractRow klassen. Den må stadig have de der felter og det skal den vel ikke. Og hvad sker der længere nede. Laver den sql baseret på contract row eller?

**Review by mrjsawdk — APPROVED** [review]
Lige en enkelt ting der mangler at blive renset (contract row) og så et spørgsmål til konsekvensen

File: Core.ApplicationServices/ExcelService.cs (hunk 2 above)
Comment (mrjsawdk) [reply Core.ApplicationServices/ExcelService.cs:142]: image

Comment (JacobMalling) [reply Core.ApplicationServices/ExcelService.cs:142]: Denne ændring du har markeret her er da netop ContractRow classen. Det er kun på ItProject der stadig findes noget der hedder Esdh nu

**Comment by mrjsawdk** [comment]
Husk at opdatere excel arket under "content"

File: Core.ApplicationServices/ExcelService.cs (hunk 1 above)
Comment (JacobMalling) [reply Core.ApplicationServices/ExcelService.cs:714]: Nej det er opbygning af excel dokument

**Pull request merged**